src/utils/parse.ts

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Tests are already submitted in https://github.com/mozilla/blurts-server/pull/3359:
+/* c8 ignore start */
+
+/** See https://en.wikipedia.org/wiki/ISO_8601 */
+export type ISO8601DateString = string | undefined | null;
+/**  See https://en.wikipedia.org/wiki/E.164 */
+export type E164PhoneNumberString = `+${string}` | null;
```

Comment:
It would be surprising for me if I'm using these types anywhere and they could suddenly also be `null` or `undefined`. If the following is causing errors elsewhere, then you probably want to use a type like `E164PhoneNumberString | null` _there_, rather than just adding it here and potentially breaking all code that references this type.

```suggestion
export type ISO8601DateString = string;
/**  See https://en.wikipedia.org/wiki/E.164 */
export type E164PhoneNumberString = `+${string}`;
```